activesupport/lib/active_support/cache.rb

```diff
@@ -139,14 +139,19 @@ def retrieve_store_class(store)
     # Some implementations may not support all methods beyond the basic cache
     # methods of +fetch+, +write+, +read+, +exist?+, and +delete+.
     #
-    # ActiveSupport::Cache::Store can store any serializable Ruby object.
+    # ActiveSupport::Cache::Store can store any serializable Ruby object. 
+    # Serializable objects must support +dump+ and +load+ operations. For example,
+    # attempting to cache HTTP ActionDispatch::Response by default will cause an error.
```

Comment:
Would it make sense to give an example that doesn't need as much context as `HTTP::Response`? If the example was trying to cache a `Proc` that may be easier for a reader to understand.

Also I think this comment makes more sense in between the two examples so that the `TypeError` example immediately follows the explanation.

Reply:
Fine with broader example.